### PR TITLE
Allow viewing ongoing queries and aborting them

### PR DESCRIPTION
**Pull request by joshenlim**
The ongoing queries that are shown here are specifically those run by the postgres user in hopes to scope down to just queries that are run via the dashboard (to prevent aborting any system queries that might be important)

image

image

### CODE REVIEW

File: apps/studio/pages/projects.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ const ProjectsPage: NextPageWithLayout = () => {
 
   return (
     <>
+      <AlertError subject="Failed to retrieve organizations" />
```
Comment (ivasilov) [inline apps/studio/pages/projects.tsx:41]: Forgotten test stuff for another feature? :)

Comment (joshenlim) [reply apps/studio/pages/projects.tsx:41]: hahahaha god dang it 🤦

File: apps/studio/components/interfaces/SQLEditor/OngoingQueriesPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+              There {queries.length === 1 ? 'is' : 'are'}{' '}
+              <span className="text-foreground-light">{queries.length}</span> quer
+              {queries.length === 1 ? 'y' : 'ies'} currently running{' '}
+              {database?.identifier !== project?.ref ? `on replica ${database?.identifier}` : ''}
+            </SheetDescription>
+          </SheetHeader>
+          <div className="max-h-full h-full divide-y overflow-y-auto">
+            {queries.length === 0 && (
+              <div className="flex items-center justify-center h-full text-foreground-light text-sm">
+                No ongoing queries running on database
```
Comment (alaister) [inline apps/studio/components/interfaces/SQLEditor/OngoingQueriesPanel.tsx:72]: A refresh button would be great somewhere in this component I reckon!

File: apps/studio/components/layouts/SQLEditorLayout/SQLEditorLayout.tsx [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+    () => (
+      <SQLEditorMenu
+        key="sql-editor-menu"
+        onViewOngoingQueries={() => setShowOngoingQueries(true)}
+      />
+    ),
+    []
+  )
+
+  console.log({ showOngoingQueries })
```
Comment (alaister) [inline apps/studio/components/layouts/SQLEditorLayout/SQLEditorLayout.tsx:24]:
```suggestion
```

**Review by alaister — APPROVED** [review]
LGTM! Couple of comments though:

File: apps/studio/components/interfaces/SQLEditor/OngoingQueriesPanel.tsx (hunk 2 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/SQLEditor/OngoingQueriesPanel.tsx:72]: yup good shout! will add that in

**Review by ivasilov — APPROVED** [review]

**Pull request merged**